schema/v1/app.cue

```diff
@@ -177,8 +177,8 @@ package v1
 }
 
 #Metrics: {
-	port: int
-	path: string
+	port: uint16 & >0 & <65536
```

Comment:
Any reason to exclude `0` and `65536`? I see we're doing it elsewhere, but I don't think we need to.

if we include them, we can switch to just `uint16` (which is [already defined](https://cuelang.org/docs/tutorials/tour/types/bounddef/) as `>=0 & <=65536`)

Reply:
Yeah I excluded those because as far as I am aware, they aren't valid port numbers.